nextcloud/server
# Hide card for apps store if disabled

How to test:

1. Login as admin
2. Go to Settings -> Admin -> Flow
3. See "More flows" card (a link to the apps store)
4. `occ config:system:set appstoreenabled --value false --type boolean`
5. Don't see "More flows" card anymore
